api/core/model_runtime/model_providers/bedrock/text_embedding/text_embedding.py

```diff
@@ -28,7 +29,7 @@
 logger = logging.getLogger(__name__)
 
 class BedrockTextEmbeddingModel(TextEmbeddingModel):
-
+    _enc = None
```

Comment:
When will the value of _enc change? Also, _enc is a class variable, not an instance variable, will there be any issues?